index.html

```diff
@@ -158,6 +162,7 @@ <h1 id="title">HRT Bus Finder</h1>
 			</div>
 		</div>
 	</script>
+                -->
```

Comment:
The above changes (or code accomplishing the same thing) were already made upstream.